Formula/readosm.rb

```diff
@@ -19,17 +19,34 @@ class Readosm < Formula
     sha256 cellar: :any, mojave:         "fcc1af52f7c13bfe4b3df0e1ca559ab79cee172c8941f51a335fb0fbb505027f"
   end
 
+  uses_from_macos "expat"
+  uses_from_macos "zlib"
+
   def install
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}"
     system "make", "install"
 
     doc.install "examples"
+
+    if OS.linux?
+      # Remove shim references
+      shim_files = [
+        doc/"examples/test_osm1",
+        doc/"examples/test_osm2",
+        doc/"examples/test_osm3",
+        doc/"examples/Makefile",
+      ]
+
+      shim_files.each do |f|
+        inreplace f, Superenv.shims_path, ""
```

Comment:
```suggestion
        inreplace f, "#{Superenv.shims_path}/", ""
```
Otherwise you leave a trailing `/` that leads to an invalid path (e.g. `$HOMEBREW_LIBRARY/shims/super/gcc` becomes `/gcc`).

Reply:
Right, I'll just open the new PR and then dispatch a Linux bottle build after it's merged.